ignite/metrics/accuracy.py

```diff
@@ -0,0 +1,45 @@
+from __future__ import division
+
+import torch
+
+from ignite.metrics.metric import Metric
+from ignite.exceptions import NotComputableError
+
+
+class Accuracy(Metric):
+    """
+    Calculates the accuracy.
+
+    - `update` must receive output of the form `(y_pred, y)`.
+    - `y_pred` must be in the following shape (batch_size, num_categories, ...)
+    - `y` must be in the following shape (batch_size, ...)
+    """
+    def reset(self):
+        self._num_correct = 0
+        self._num_examples = 0
+
+    def update(self, output):
+        y_pred, y = output
```

Comment:
Maybe we need also to compare shapes of `y_pred` and `y` and raise error if they are different ?

Reply:
I was planning on adding that!

Reply:
resolved. code is verbose, let me know if you need it trimmed down.